core/extraction/towns_fund.py

```diff
@@ -547,6 +549,26 @@ def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict) -> pd.Dat
 
     df_funding = convert_financial_halves(df_funding, "Reporting Period")
     df_funding.reset_index(drop=True, inplace=True)
+    if list(project_lookup.values())[0].split("-")[0] == "HS":
+        # mask for unused funding cells
+        unused_fhsf_mask = df_funding.loc[
+            # drop unused FHSF Questions
+            (
+                df_funding["Funding Source Name"].isin(
+                    [
+                        "Town Deals 5% CDEL Pre-Payment",
+                        "Towns Fund RDEL Payment which is being utilised on TF project related activity",
+                        "How much of your RDEL forecast is contractually committed?",
+                    ]
+                )
+            )
+            |
+            # drop unused FHSF cells
+            (df_funding["Secured"].isna())
+            & ((df_funding["Start_Date"] > datetime(2023, 10, 1)) | (df_funding["Start_Date"].isna()))
+        ]
+        # TODO: Remove hard coded date
+        df_funding = df_funding.drop(unused_fhsf_mask.index)
```

Comment:
A user only uploads for either TD or FHSF. In the case where they have selected their fund_type as FHSF, cells pertaining to this financial information are greyed out. Rather than dropping this information after collecting it, it might make more sense to not collect it in the first place?

Reply:
In this case, that might be more difficult than what you've done, or more expensive; if that is the case then dropping might be easier :)

Reply:
I did think about this. I think might be quite a bit harder, since we use iloc to grab the information of the page in the first instance, but we need to drop data based on some very specific rules, that would be hard to apply there.